docs/disaster_picker.js

```diff
@@ -1,21 +1,31 @@
-import {getDisaster, getDisasters} from './resources.js';
+import {getDisasters} from './firestore_document.js';
+import {getDisaster} from './resources.js';
 
 export {initializeDisasterPicker};
 
-/** Initializes the disaster picker. */
-function initializeDisasterPicker() {
-  const disasterDropdown = document.getElementById('disaster-dropdown');
-  getDisasters().forEach((disaster) => {
-    const disasterItem = document.createElement('option');
-    disasterItem.innerHTML = disaster;
-    if (disaster === getDisaster()) {
-      disasterItem.selected = 'selected';
-    }
-    disasterDropdown.appendChild(disasterItem);
-  });
+/**
+ * Initializes the disaster picker.
+ * @param {Promise} firebaseAuthPromise Promise that completes when logged in to
+ *     Firebase
+ */
+function initializeDisasterPicker(firebaseAuthPromise) {
+  firebaseAuthPromise.then(() => {
+    const disasterDropdown = document.getElementById('disaster-dropdown');
```

Comment:
jquery for consistency? Also the method createOptionFrom exists if you're interested.

Reply:
Done on jQuery. Depending on add_disaster.js from this seems weird, and I don't want to refactor it out of add_disaster.js because it would mess with your PR (also the selects there have a bit more structure than the ones here need).